test/runBTagAnalyzer_cfg.py

```diff
@@ -473,7 +473,7 @@
         postfix = postfix
     )
     getattr(process,'patJetPartonMatchCA8'+postfix).matched = cms.InputTag(genParticles)
```

Comment:
Yes, CA8 should be switched to AK8 jets. Is somebody handling it?
BTW, I cannot push for some reason, but I can try developing on my fork.